components/content-service-api/workspace.proto

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2020 TypeFox GmbH. All rights reserved.
+// Licensed under the GNU Affero General Public License (AGPL).
+// See License-AGPL.txt in the project root for license information.
+
+syntax = "proto3";
+
+package contentservice;
+
+option go_package = "github.com/gitpod-io/gitpod/content-service/api";
+
+service WorkspaceService {
+    // DownloadUrlWorkspace provides a URL from where the content of a workspace can be downloaded from
+    rpc DownloadUrlWorkspace(DownloadUrlWorkspaceRequest) returns (DownloadUrlWorkspaceResponse) {};
```

Comment:
The other services use a more succinct name for this operation:

```suggestion
    rpc DownloadURL(DownloadURL) returns (DownloadURL) {};
```

Reply:
Good point. Had this all in one service at first. That's why a different name was required. That's not the case anymore.

Reply:
Having the same operation name seems not to work:

```
$ ./generate.sh 
go: found github.com/golang/protobuf/protoc-gen-go in github.com/golang/protobuf v1.3.5
workspace.proto:20:12: "contentservice.DownloadUrlRequest.owner_id" is already defined in file "blobs.proto".
workspace.proto:19:9: "contentservice.DownloadUrlRequest" is already defined in file "blobs.proto".
workspace.proto:24:12: "contentservice.DownloadUrlResponse.url" is already defined in file "blobs.proto".
workspace.proto:23:9: "contentservice.DownloadUrlResponse" is already defined in file "blobs.proto".
workspace.proto:28:12: "contentservice.DeleteRequest.owner_id" is already defined in file "blobs.proto".
workspace.proto:27:9: "contentservice.DeleteRequest" is already defined in file "blobs.proto".
workspace.proto:32:9: "contentservice.DeleteResponse" is already defined in file "blobs.proto".
workspace.proto:13:21: "contentservice.DownloadUrlRequest" seems to be defined in "blobs.proto", which is not imported by "workspace.proto".  To use it here, please add the necessary import.
...
```

Reply:
Then maybe `DownloadWorkspaceURL(Request|Response)`?